.gitignore

```diff
@@ -14,7 +14,12 @@ htmlcov
 *.pdf
 
 .vscode*
+__pycache__
+.coverage
 
 # cache/dump
 *.json
 *.yaml
+
+
+PyDataConfig.egg-info/
```

Comment:
Could you remove the .gitignore changes? Best not to have these in feature specific PRs like this.